cilium_test.go

```diff
@@ -0,0 +1,263 @@
+package sonobuoy_plugin
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/url"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	appv1alpha1 "github.com/giantswarm/apiextensions/v3/pkg/apis/application/v1alpha1"
+	"github.com/giantswarm/apiextensions/v3/pkg/apis/release/v1alpha1"
+	"github.com/giantswarm/apiextensions/v3/pkg/label"
+	"github.com/giantswarm/backoff"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	v1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/remotecommand"
+	capiv1alpha3 "sigs.k8s.io/cluster-api/api/v1beta1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/giantswarm/sonobuoy-plugin/v5/pkg/capiutil"
+	"github.com/giantswarm/sonobuoy-plugin/v5/pkg/ctrlclient"
+)
+
+const (
+	ciliumAppName     = "cilium"
+	ciliumDsName      = "cilium"
+	ciliumDsNamespace = "kube-system"
+)
+
+// Test_Cilium test the cilium app comes up healthy.
+func Test_Cilium(t *testing.T) {
+	t.Parallel()
+
+	var err error
+
+	ctx := context.Background()
+
+	tcCtrlClient, err := ctrlclient.CreateTCCtrlClient()
+	if err != nil {
+		t.Fatalf("error creating TC k8s client: %v", err)
+	}
+
+	cpCtrlClient, err := ctrlclient.CreateCPCtrlClient()
+	if err != nil {
+		t.Fatalf("error creating CP k8s client: %v", err)
+	}
+
+	regularLogger, err := micrologger.New(micrologger.Config{})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	logger := NewTestLogger(regularLogger, t)
+
+	clusterID, exists := os.LookupEnv("CLUSTER_ID")
+	if !exists {
+		t.Fatal("missing CLUSTER_ID environment variable")
+	}
+
+	// Get Cluster.
+	var cluster *capiv1alpha3.Cluster
```

Comment:
```suggestion
	var cluster *capiv1alpha3.Cluster
```

This should be v1beta1 by now

Reply:
It is. Forgot to rename the import alias. fixed